index.js

```diff
@@ -43,4 +60,28 @@ function toggleColor(element) {
 
 /***** Deliverable 2 *****/
 
+const playerForm = document.querySelector("#new-player-form")
+
+playerForm.addEventListener("submit", function(event) {
+  event.preventDefault()
+
+  const number = event.target.number.value
+  const name = event.target.name.value
+  const nickname = event.target.nickname.value
+  const photo = event.target.photo.value
+
+  const player = {
+    number: number,
+    name: name,
+    nickname: nickname,
+    photo: photo,
+    likes: 0
+  }
```

Comment:
Looks good! If you want to tighten the code up a bit, you could create your object with the values from the input fields directly (just a stylistic choice): 

```js
  const player = {
    number: event.target.number.value,
    name: event.target.name.value,
    nickname: event.target.nickname.value,
    photo: event.target.photo.value,
    likes: 0
  }
```